docs/getting-started.md

```diff
@@ -45,23 +45,23 @@ So all you need is:
 1. Download KICS binaries based on your OS
 1. Extract files
 1. Run kics executable with the cli options as described below (note that kics binary should be located in the same directory as queries directory)
-   ```
-   ./kics scan -p <path-of-your-project-to-scan> -o <output-results.json>
+   ```shell
+   ./kics scan -p '<path-of-your-project-to-scan>' -o '<output-results.json>'
```

Comment:
I was thinking about this example, shouldn't we update it for -o flag? I mean, this still works, but I think it can confuse user, since it can be used to generate multiple result types...